### PR TITLE
Restore missions' barbed wire

### PR DESCRIPTION
**Pull request by JovialFeline**
This is a riveting follow-up to https://github.com/OpenRA/OpenRA/pull/21373.

https://github.com/user-attachments/assets/503fed59-ad0b-4e28-9050-36f7b6c2daaa

With the importer's help, I went back and restored barbed wire (FENC) for missions put together before that PR. The list:
  - `allies-05a`, Tanya's Tale (A)
  	- Only two added; others were already placed.
  - `allies-05b` & `allies-05c`, Tanya's Tale (B & C)
  - `allies-10a`, Suspicion
  - `fall-of-greece-1-personal-war`
  - `fall-of-greece-2-evacuation`
  	- I opted to make Neutral the fence's owner instead of nearby England, to make it less fiddly to break if that is the player's wish.
  - `siberian-conflict-3-wasteland`
  - `soviet-01`, Lesson in Blood
  - `soviet-09`, Liability Elimination
  - `soviet-10`, Overseer

Other maps already have the barbed wire placed by hand:
  - `allies-01`, In the Thick of It
  - `allies-02`, Five to One
  - `allies-03a` & `allies-03b`, Dead End
    - The latter has some wire shifted to avoid a cliff corner, but all 128 bits seem to be there.
  - `allies-09a`, Extract Kosygin
  - `sarin-gas-3-controlled-burn`
    - The fencing here is a little different, partly to protect a new Radar Dome in the southwest corner.
  - `in-the-nick-of-time`
  - `negotiations`
  - `soviet-02a`, Thin Red Line
  - `soviet-03`, Covert Cleanup

`allies-06a` fencing is covered by https://github.com/OpenRA/OpenRA/pull/21401, and `allies-06b` will follow.

VolkovNChitzkoi_WeirdFence
I skipped the single FENC in `soviet-soldier-volkov-n-chitzkoi` because it seems like a mapper's goof.

Near as I can tell, the only original TD mission with placed BARB is Hostile Takeover, which remains on the to-do list. Mousetrap is the only RA mission to feature BARB, where it's placed to resemble power coils.

### CODE REVIEW

File: mods/ra/maps/soviet-01/map.yaml [hunk 1]
```diff
@@ -586,6 +586,78 @@ Actors:
 	CivSpawn: waypoint
 		Location: 55,65
 		Owner: Neutral
+	Actor157: fenc
+		Location: 34,76
+		Owner: USSR
```
Comment (PunkPun) [inline mods/ra/maps/soviet-01/map.yaml:591]: why are these owned by USSR? You don't wanna waste yak ammo on them

Comment (JovialFeline) [reply mods/ra/maps/soviet-01/map.yaml:591]: I'm confused by the question since the player is also USSR.

Comment (PunkPun) [reply mods/ra/maps/soviet-01/map.yaml:591]: interesting

**Review by RoosterDragon — APPROVED** [review]
Checked all the maps, and all the new fences look sensible.

**Review by PunkPun — APPROVED** [review]
LGTM

**Comment by PunkPun** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/a1a5cb80481a8c79760a68ac7c053dd8e258a05c...c82277937c62ce29779fd9a120298eced8eda7df)

**Pull request merged**